requirements.txt

```diff
@@ -65,6 +65,7 @@ mypy-extensions==0.4.3
 nose==1.3.7
 nose2==0.9.1
 numpy==1.17.4
+-e git+git@github.com:panther-labs/panther-cli.git@0a4ff06aaa1bf9c38975222941a7207675114feb#egg=panther_cli
```

Comment:
Do we need this? Is this library published somewhere that would be installed on a fresh clone of this repo?

Reply:
I don't actually know what this line means, not sure what added it. Let me look into it.